go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/open-policy-agent/opa-envoy-plugin
 
-go 1.20
+go 1.21
+
+toolchain go1.21.0
 
 require (
 	github.com/envoyproxy/go-control-plane v0.12.0
 	github.com/golang/protobuf v1.5.4
-	github.com/open-policy-agent/opa v0.62.1
+	github.com/open-policy-agent/opa v0.63.0
```

Comment:
Interesting https://github.com/open-policy-agent/opa-envoy-plugin/pull/536/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L8. The PR raised by dependabot does not increase the min go version.